digiroad2-geo/src/test/scala/fi/liikennevirasto/digiroad2/linearasset/DamagedByThawFillerSpec.scala

```diff
@@ -0,0 +1,86 @@
+package fi.liikennevirasto.digiroad2.linearasset
+
+
+import fi.liikennevirasto.digiroad2.{Point}
+import fi.liikennevirasto.digiroad2.asset.LinkGeomSource.NormalLinkInterface
+import fi.liikennevirasto.digiroad2.asset._
+import fi.liikennevirasto.digiroad2.linearasset.LinearAssetFiller.{ChangeSet}
+import org.joda.time.DateTime
+import org.joda.time.format.DateTimeFormat
+import org.scalatest.{FunSuite, Matchers}
+
+
+class DamagedByThawFillerSpec extends FunSuite with Matchers {
+
+  object damagedByThawFiller extends DamagedByThawFiller
+
+  private val dateFormat = "dd.MM.yyyy"
+  private val formatter = DateTimeFormat.forPattern(dateFormat)
+
+  def dateToString(date: DateTime): String = {
+    date.toString(dateFormat)
+  }
```

Comment:
This method have impact on the tests, please use the one created on DamagedByThawFiller.scala